egov/egov-collectionweb/src/main/java/org/egov/collection/web/actions/receipts/ChequeRemittanceAction.java

```diff
@@ -205,12 +206,29 @@ public String listData() {
             fundQuery.setString("accountNumberId", accountNumberId);
             List<String> fundCodeList = fundQuery.list();
             final String fundCode = fundCodeList != null && !fundCodeList.isEmpty() ? fundCodeList.get(0).toString() : null;
+            Long dateInLongFromDate=null;
+            Long dateInLongToDate=null;
 
+           
             final CFinancialYear financialYear = financialYearDAO.getFinancialYearById(finYearId);
+            DateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
+            Date dateFrom;
+            Date toDates;
+            try {
+                dateFrom = formatter.parse(financialYear.getStartingDate().toString());
```

Comment:
Consider from date and to date and use dates from financial year if from date and to date is null.